### ISSUE TITLE
Multiple sets of comments for initial view of synthesis tree-view

### ISSUE THREAD

**Issue by jimallman** [OpenTreeOfLife/opentree#592]
I noticed this while working on caching these comments, but I'm pretty sure this is not a new problem. It seems the "location" for this initial view resolves in either of two ways, possibly depending on the incoming URL?

1. based on the default URL `/opentree`, which fetches comments like so:
```
https://api.github.com/search/issues?q=%22URL+%7C+%2Fopentree+%22+repo:OpenTreeOfLife%2Ffeedback+state:open&sort=created&order=desc
```

2. based on tree and node IDs, which retrieves a different set of comments:
```
https://api.github.com/search/issues?q=%22Synthetic+tree+node+id+%7C+132+%22+%22Synthetic+tree+id+%7C+otol.draft.22+%22+repo:OpenTreeOfLife%2Ffeedback+state:open&sort=created&order=desc
```

Possibly based on whether or not we redirect? But we **always** redirect... Will investigate.

**Comment by jimallman** [comment]
To clarify, the URLs above can be passed directly to cURL to see the comments in each case.

**Comment by jimallman** [comment]
I believe this has been addressed already by 03978363734dbd28c69ebff98c8a7da9a0608511. Please feel free to re-open if the symptom re-appears!
